src/host/_stream.cpp

```diff
@@ -360,6 +357,9 @@ constexpr unsigned int LOCAL_BUFFER_SIZE = 100;
         coordScreenBufferSize.X = textBuffer.GetLineWidth(CursorPosition.Y);
     }
 
+    static constexpr unsigned int LOCAL_BUFFER_SIZE = 1024;

```

Comment:
I had a hard time choosing a good number. 256 was faster for shorter lines, 1024 was optimally fast for longer lines (no improvements beyond 1024) and 512 was basically the worst of both worlds.

Reply:
> if accessibility events are enabled

Wait, is it *enabled* or *disabled*?

Reply:
Yeah the improvements are larger if a11y events are enabled.